pkg/nodeconfig/nodeconfig.go

```diff
@@ -356,6 +356,38 @@ func (nc *nodeConfig) configureCNI() error {
 	return nil
 }
 
+// ErrWriter is a wrapper to enable error-level logging inside kubectl drainer implementation
+type ErrWriter struct {
+	log logr.Logger
+}
+
+func (ew ErrWriter) Write(p []byte) (n int, err error) {
+	// log error
+	ew.log.Error(err, string(p))
+	return len(p), nil
+}
+
+// OutWriter is a wrapper to enable debug-level logging inside kubectl drainer implementation
+type OutWriter struct {
+	log logr.Logger
+}
+
+func (ow OutWriter) Write(p []byte) (n int, err error) {
+	// log debug
+	ow.log.V(1).Info(string(p))
```

Comment:
Ideally we need a INFO log when we are draining and evicting pods

Reply:
I would like to see a log when we are draining and evicting pods and not for all situations. How that is achieved is up to you.

Reply:
Fixed in https://github.com/openshift/windows-machine-config-operator/pull/676